02.24.16

manitghogar opened the pull request.

manitghogar committed:
Reading and Writing Files

Learnt how to shorten code to write and read files

manitghogar committed:
Learn how to read and write files

Pass two files to the python script and overwrites the second file with
the information contained in the first file.

manitghogar committed:
Functions!

Learning about functions and how to pass arguments to them in the same
way we pass arguments of argv to our scripts

manitghogar committed:
Learning to write functions

Giving functions its arguments in different ways. Wrote my own function
and found 7 ways to pass it arguments. Still three more ways that I
have not thought of. Will come back to this one.

manitghogar committed:
Functions and Files

Learning how to pass files to functions and execute functions on these
files

manitghogar committed:
Functions and variables

Learnt to build functions and then assign the outcomes of these
functions to variables which could then be used within formulas.

arjunblj commented on 21/ex21.py:
```diff
@@ -0,0 +1,59 @@
+age = add(40,5)
+height = subtract(80,4)
+weight = multiply(90,2)
+iq = divide(110,2)Computer2241
```
What is `Computer2241`?

manitghogar replied:
Didn't even notice that, had to type that somewhere else was probably still
stuck on this screen

On Wed, Feb 24, 2016 at 5:21 PM, Arjun Balaji notifications@github.com
wrote:

> In 21/ex21.py
> https://github.com/manitghogar/lpthw/pull/5#discussion_r54017180:
> 
> > -   return a - b
> >   +#multiply two arguments passed to the fucntion.
> >   +def multiply(a, b):
> > -   print "MULTIPLYING %d \* %d" % (a, b)
> > -   return a \* b
> >   +#divides the two arguments passed ot the function
> >   +def divide(a, b):
> > -   print "DIVIDING %d / %d" % (a, b)
> > -   return a / b
> >   +
> >   +print "Let's do some math with just functions!"
> >   +#we assign the outcoms of the above functions to variables so that they can be used in additional functions.
> >   +age = add(40,5)
> >   +height = subtract(80,4)
> >   +weight = multiply(90,2)
> >   +iq = divide(110,2)Computer2241
> 
> What is Computer2241?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/manitghogar/lpthw/pull/5/files#r54017180.

manitghogar committed:
Additional practice on escape sequences, format characters, variables and functions.

Revision of the previous exercises especially on assigning function outputs to variables and using variabls as arguments in the function.

manitghogar committed:
Practicing pure functions

Learn how we can incorporate functions into other functions. Still
confused about the pop() function. IT does not exactly remove the word
that I expect it to remove.

manitghogar committed:
Take the test

Testing on exercises 24 and 25.

manitghogar committed:
Boolean Logic

Learn different equality/comparison operators.

manitghogar committed:
If Statements

Implemented Boolean logic into if statements
